Library/Formula/collectd.rb

```diff
@@ -18,25 +29,16 @@ class Collectd < Formula
     depends_on "autoconf" => :build
   end
 
-  stable do
-    url "https://collectd.org/files/collectd-5.5.0.tar.bz2"
-    mirror "http://pkgs.fedoraproject.org/repo/pkgs/collectd/collectd-5.5.0.tar.bz2/c39305ef5514b44238b0d31f77e29e6a/collectd-5.5.0.tar.bz2"
-    sha256 "847684cf5c10de1dc34145078af3fcf6e0d168ba98c14f1343b1062a4b569e88"
-
-    patch do
-      url "https://github.com/collectd/collectd/commit/e0683047a42e217c352c2419532b8e029f9f3f0a.diff"
-      sha256 "7053170a072d27465b69eed269d32190ec810bcb0db59f139a1682e71a326fdd"
-    end
-  end
-
   # Will fail against Java 1.7
   option "with-java", "Enable Java 1.6 support"
+  option "with-riemann", "Enable write_riemann support"
   option "with-debug", "Enable debug support"
 
   deprecated_option "java" => "with-java"
   deprecated_option "debug" => "with-debug"
 
   depends_on "pkg-config" => :build
+  depends_on "protobuf-c" if build.with? "riemann"
```

Comment:
Can you use `:optional` here and fill in the `option "with-protobuf-c", "Enable write_riemann support"` or similar? Thanks!